package.json

```diff
@@ -356,45 +356,67 @@
                     "description": "Run the linter on active files."
                 },
                 "julia.lint.missingrefs": {
-                    "type": "boolean",
-                    "default": true,
-                    "description": "Report possibly missing references."
+                    "type": "string",
+                    "default": "all",
+                    "enum": [
+                        "none",
+                        "getfield missing",
+                        "all"
+                    ],
+                    "description": "Highlight unknown symbols. The `getfield missing` option will not mark unknown fieldnames."
```

Comment:
Is this right? I would have thought that the option `getfield missing` would _mark_ the cases where I'm accessing a field that doesn't exist?

Not for this release, but should we maybe just split this into two bool options, i.e. have just have one `bool` option that enables the `f.a` detection stuff?